tests/mypy_test.py

```diff
@@ -18,6 +18,8 @@
 import sys
 import toml
 import tempfile
+from glob import glob
+from pathlib import Path
 from typing import Dict, NamedTuple
 
 PY2_NAMESPACE = "@python2"
```

Comment:
This doesn't seem to be worth keeping in a global constant. It's already hard-coded in a couple places.